src/pages/MyOffersPage.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Paper } from "@material-ui/core";
+import CenteredComponent from "../components/HomePage/CenteredComponent";
+import { MainMask } from "../components/HomePage/MainMask";
+
+export const MyOffersPage = () => (
+    <>
+        <MainMask />
+        <CenteredComponent>
+            <Paper style={{ width: "60%", padding: "24px 72px", boxSizing: "content-box" }}>
```

Comment:
We should use a `<Card>` instead of a `<Paper>`. Also, it is a bad practice to have a style object applied directly like so (https://marmelab.com/blog/2017/02/06/react-is-slow-react-is-fast.html#beware-of-object-literals-in-jsx) (I know it is being done in other places, but it should be fixed as well!!!)

Please use the `useStyles` approach.